modules/render.js

```diff
@@ -1,5 +1,6 @@
-/* eslint-disable no-undef */
-/* eslint-disable no-unused-vars */
+import Storage from './storage.js';
+import Book from './books.js';
+
 class Render {
   static displayLibrary() {
```

Comment:
- [ ] it would be better if you use the es6 `arrow function` instead of the `normal function` in order to practice to work with both